Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -1087,10 +1087,29 @@ std::unique_ptr<ROMol> molzip(std::vector<ROMOL_SPTR> &decomposition,
     }
   }
 
+  if(decomposition.empty())
+    return nullptr;
```

Comment:
```suggestion
  if(decomposition.empty()) {
    return nullptr;
  }
```
Since you regularly do this, maybe you should run clang-tidy across your PRs before submitting them? It has a rule for this: readability-braces-around-statements

Reply:
I'll add that.  This was burned into me with the OpenEye style guide where Roger said "that extra bracket slows down the compiler"